deepface/modules/kyc_validation.py

```diff
@@ -0,0 +1,148 @@
+import base64
+import io
+import os
+import random
+from typing import List
+import asyncio
+
+
+import cv2
+from deepface import DeepFace
+import numpy as np
+
+detector_backend_extract = "retinaface"
```

Comment:
detector names are being set in extract_faces, we should use the one set instead of static one